### PR TITLE
Ehrenfest can be set up and customized via a new lionml namelist.

### PR DESCRIPTION
**Pull request by ramirezfranciscof**
As it says in the title.

### CODE REVIEW

**Comment by fedepedron** [comment]
Maybe adding a Ehrenfest test would be nice; I already tested everything, except for intel compilation and Ehrenfest runs.

**Comment by Nick-Chaos** [comment]
tested with intel compilers, single point and ECP calculations

**Comment by ramirezfranciscof** [comment]
Thanks @Nick-Chaos ! Ehrenfest test comig soon...

**Comment by ramirezfranciscof** [comment]
Made some minor corrections and changes. I think it would be better not to squash this PR but create a merge commit instead.

Ehrenfest test next week, I promise.

**Comment by ramirezfranciscof** [comment]
@fedepedron

**Comment by ramirezfranciscof** [comment]
There was a problem with the scripts calling the tests and the obligatory restart at the end of the run. The first I corrected and the second I leave commented for another time, but there is also an issue with the forces not being equal in BO and EH. This is not strictly incorrect since I had to put very high initial velocities for the change in positions to be noticed, and velocities affect the forces in EH.

I'm leaving the test so that people can test that ehrenfest code actually runs (it says "doing ehrenfest" in stdo), but still there is no point in no point in comparing with BO and hence no way of checking if it is running *correctly* yet...

**Comment by fedepedron** [comment]
Tested with LIO/GROMACS, with/without AINT and full CPU. However, I cannot still test Ehrenfest due to some problems with amber. I'll try a couple of things.

File: testamber/badamber/gpu_options [hunk 1]
```diff
@@ -0,0 +1,5 @@
+little_cube_size 10.0
```
Comment (fedepedron) [inline testamber/badamber/gpu_options]: This file is no longer needed, options should be included in the *.in file.

File: testamber/watersymbo/basis_dzvp.in [hunk 2]
```diff
@@ -0,0 +1,282 @@
+gaussian
```
Comment (fedepedron) [inline testamber/watersymbo/basis_dzvp.in]: Using int_basis instead?

File: testamber/watersymbo/gpu_options [hunk 3]
```diff
@@ -0,0 +1,5 @@
+little_cube_size 10.0
```
Comment (fedepedron) [inline testamber/watersymbo/gpu_options]: Same as above.

**Pull request merged**